### DISCUSSION TITLE
Unable to render HTML pages with HTML comments inside of them

### DISCUSSION THREAD

**Discussion by ZaneStauffer** [requarks/wiki#3815] [Error / Bug Report]
Pages created using the HTML editor (in Version `2.5.201`) are unable to render if they have HTML comments inside of them (example: `<!-- Comment -->` ). The program outputs the following error:

![image](https://user-images.githubusercontent.com/38232438/114299716-dbe03380-9a8a-11eb-806e-026ee55c4c44.png)

# Reproducing

1. Create a new page.
2. Select the Code Raw HTML editor.
3. Place an HTML comment anywhere in the page's markdown.  `<!-- Comment -->`
4. Create the page. It will be unable to render and will display a duplicate page error.

**Comment by lkho** [comment]
related issue: https://github.com/Requarks/wiki/discussions/3709#discussioncomment-596783
related PR: https://github.com/Requarks/wiki/pull/3823
